crates/types/src/nodes_config.rs

```diff
@@ -48,6 +48,7 @@ pub enum Role {
     MetadataStore,
     /// [IN DEVELOPMENT] Serves a log server for replicated loglets
     LogServer,
+    Ingress,
```

Comment:
Maybe better to name this `HttpIngress` in case we ever split the Kafka ingress?

Reply:
Will update it.